app/styles/get-started.css

```diff
@@ -0,0 +1,70 @@
+.landing {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  height: 80vh;
+  width: 70%;
+  margin: auto;
+}
+
+.landing__mainHeading {
+  font-size: 2rem;
+  color: black;
+}
+
+.landing__subHeading {
+  font-size: 1.5rem;
+  color: black;
+}
```

Comment:
Use `em` instead of `rem`.
`em` helps in scaling the whole BEM's block with just `font-size` change.